web-app/src/components/HolidayModal/index.js

```diff
@@ -59,12 +71,40 @@ const HolidayModal = ({
             </h2>
             <h4>Duration</h4>
           </Stat>
-        </StatWrap>
-        {shouldShowAdminControls() && (
-          <ButtonWrap>
-            <Button label="Approve" onClick={approveHoliday} />
-            <Button label="Reject" onClick={rejectHoliday} />
-          </ButtonWrap>
+        </FlexWrap>
+        {shouldShowAdminControls() &&
+          (toggled === false ? (
```

Comment:
Try and follow the way I mentioned above rather than doing ternaries, it'll look cleaner.